lib/elixir/test/elixir/module_test.exs

```diff
@@ -50,11 +50,17 @@ defmodule ModuleTest do
   end
   Module.eval_quoted __MODULE__, contents, [], file: "sample.ex", line: 13
 
+  defp purge(modules) do
+    Enum.each modules, fn(m) ->
```

Comment:
Seems premature use of `Enum` here, in places where we call `purge/1` helper we pass only one module.